thermalnetwork/tests/test_base.py

```diff
@@ -52,7 +52,7 @@ def setUp(self) -> None:
         self.test_outputs_path.mkdir(exist_ok=True)
 
         # Save the original borehole length and number of boreholes, so they can be restored after the tests run.
-        sys_param_1_ghe = json.loads((self.system_parameter_path_1_ghe).read_text())
+        sys_param_1_ghe = json.loads(self.system_parameter_path_1_ghe.read_text())
```

Comment:
We need to read the text first or else `loads` gets whiny

Reply:
Isn't that what is happening here?

Reply:
Sorry, I wasn't clear. Yes, the way I have it does exactly that.

Reply:
I also wasn't clear. In the original, the parentheses seem redundant, but apparently they are doing _something_???

Reply:
There were some redundant parentheses in the original, but at least they weren't stopping it from working. It's cleaned up now.